docs/config.json

```diff
@@ -358,6 +358,11 @@
               "title": "Terraform",
               "slug": "/access-controls/login-rules/terraform/",
               "forScopes": ["enterprise", "cloud"]
+            },
+            {
+              "title": "Kubernetes",
```

Comment:
Maybe it's better to include the operator in the name, otherwise it will be confusing